src/config/dynamic-theme-fixes.config

```diff
@@ -18420,6 +18420,7 @@ INVERT
 .mw-kartographer-map
 .mw-kartographer-mapDialog-map
 .mw-logo-tagline
+.mw-logo-icon
```

Comment:
that will need to be changed

maybe for en.wiktionary it will work but on pl.wiktionary I have this logo and it should not be inverted:
![obraz](https://github.com/user-attachments/assets/a80b7bd1-18c3-44ec-a0e7-cfb0c3a0a207)

So as a resolve I propose `img[src="/static/images/icons/enwiktionary.svg"]`

Reply:
that works for me